lib/toc_obj.js

```diff
@@ -13,11 +13,26 @@ const getId = ({ attribs = {}, parent }) => {
   return attribs.id || (!parent ? '' : getId(parent));
 };
 
+/**
+ * Identify a heading that to be unnumbered or not.
+ *
+ * @param {Node} {attribs.class} string contains all classes of the heading.
+ * @param {Set<string>} searchClassSet classes to search.
+ * @returns bool
+ */
+const isUnnumbered = ({ attribs = { class: "" } }, searchClassSet) => {
+  let classList = attribs.class;
+  return classList &&
+    classList.split(" ").filter(item => searchClassSet.has(item)).length > 0;
+};
```

Comment:
IMHO passing the `unnumbered` through a `className` is kinda bad. What about using a `data-*` attribute? Like `<h1 class="" data-toc-unnumbered="false">`

Reply:
That's a good idea. I will make a change later.